src/css-token-stream-value.js

```diff
@@ -16,16 +16,32 @@
   }
   internal.inherit(CSSTokenStreamValue, CSSStyleValue);
 
-  CSSTokenStreamValue.prototype.keys = function() {
-    return internal.objects.iterator(this._listOfReferences, function(key, value) { return key; });
+  CSSTokenStreamValue.prototype[Symbol.iterator] = function() {
```

Comment:
what's the basic support level for the polyfill? I notice that `Symbol` isn't used anywhere else, and it's from ES6.